src/frontend/js/types/Joanie.ts

```diff
@@ -250,6 +250,12 @@ export interface NestedCertificateOrder extends AbstractNestedOrder {
   course: undefined;
   enrollment: EnrollmentLight;
 }
+export const isNestedCredentialOrder = (
+  obj: NestedCertificateOrder | NestedCredentialOrder,
+): obj is NestedCredentialOrder => {
+  return !obj.enrollment && !!obj.course;
+};
+
```

Comment:
I wonder if we should not move those type guards in a dedicated module. Surely this has not to be adressed in this PR.

Reply:
I can be a idea.
I like to have them close to the type they check.

But if (when) we switch to the generated client, they'll have to have they own file anyway